inc/file.class.php

```diff
@@ -144,6 +144,10 @@ public function prepareInputForAdd($input) {
          $input['name'] = $actualFilename;
       }
 
+      if (!isset($input['comment']) || empty($input['comment'])) {
+         $input['comment'] = '';
+      }
+
```

Comment:
I think this block is not necessary. If comment is not populated, then GLPI will not try to update it.